### PR TITLE
Update translations in GitHub actions

### PR DESCRIPTION
**Pull request by fmoc**
Right now, translations are updated on Drone. This PR moves the task to GitHub actions.

### CODE REVIEW

File: .github/workflows/translate.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        run: docker run --rm -it -e TX_TOKEN -v (readlink -f .):/ws rabits/qt:5.12-desktop bash -c "cd /ws/translations && make l10n-read"
+
+      - name: l10n-push
+        run: cd translations && make l10n-push
+
+      - name: l10n-pull
+        run: cd translations && make l10n-pull
+
+      - name: l10n-write
+        run: docker run --rm -it -e TX_TOKEN -v (readlink -f .):/ws owncloudci/transifex:latest bash -c "cd /ws/translations && make l10n-write"
```
Comment (TheOneRing) [inline .github/workflows/translate.yml:30]: `TX_TOKEN=${{ secrets.TX_TOKEN }}`

Comment (TheOneRing) [reply .github/workflows/translate.yml:30]: And maybe we can also just kill the stupid make file and call the stuff from here?

Comment (fmoc) [reply .github/workflows/translate.yml:30]: I injected the secret as an environment variable once for the entire job, like in the Drone job. I just need to forward the value here. I'm not sure the value isn't used elsewhere as well.

Comment (TheOneRing) [reply .github/workflows/translate.yml:30]: @xoxys any idea?

Comment (xoxys) [reply .github/workflows/translate.yml:30]: No idea

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**